websocket-native/src/main/java/org/ballerinalang/net/websocket/WebSocketUtil.java

```diff
@@ -251,8 +250,8 @@ public static BError createErrorByType(Throwable throwable) {
             errorCode = WebSocketConstants.ErrorCode.WsInvalidHandshakeError.errorCode();
         } else if (throwable instanceof IOException) {
             errorCode = WebSocketConstants.ErrorCode.WsConnectionError.errorCode();
-            cause = createErrorCause(throwable.getMessage(), IOConstants.ErrorCode.GenericError.errorCode(),
-                    IOConstants.IO_PACKAGE_ID);
+            cause = createErrorCause(throwable.getMessage(), WebSocketConstants.ErrorCode.WsGenericError.
+                    errorCode(), ModuleUtils.getWebsocketModule());
```

Comment:
@Bhashinee, why we are changing this to the WebSocket level error?. Why can't we replace those constants like this(https://github.com/ballerina-platform/ballerina-standard-library/issues/926)

Reply:
These errors are not related to the Ballerina IO package. Those are just Java IO exceptions that are thrown because of WebSocket connection issues. So I think it is not good to create them as Ballerina IO package errors.